specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```
@@ -99,7 +99,7 @@
   "paths": {
     "/dataRegistries": {
       "get": {
-        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe endpoint of Data Registry APIs is bound to the region of your maps account. Data Registry Service is not available for global maps account.\n\nThis API allows the caller to fetch the list of all data registries registered previously using the [Register or Replace API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/register-or-replace).\n\nThis API won't check whether Azure Maps can access the underlying content and whether integrity check is passed for each data registry. Please use [Get Data Registry API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/get-data-registry) for the latest status individually.\n\n## Submit List Request\n\nTo list all your map data content you will issue a `GET` request with no additional parameters.\n\n## List Response\n\nThe Data Registry List API returns the complete list of all data registries in `json` format.",
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe `List` request returns an array containing all data registries previously registered using the [Register or Replace API](https://docs.microsoft.com/rest/api/maps/data-registry/register-or-replace).\n\nThis API won't check whether Azure Maps can access the underlying content and whether integrity check is passed for each data registry. Please use [Get Data Registry](https://docs.microsoft.com/rest/api/maps/data-registry/get-data-registry) for the latest status individually.\n\nThe data registry service endpoint is limited in scope to the region of your Azure Maps account and is not available for global Azure Maps accounts.\n\n## Submit List Request\n\nTo list all your data registries, issue a `GET` request with no additional parameters.\n\n## List Response\n\nThe `List` request returns an array of data registries in `JSON` format in the body of the response.",
```

```suggestion
        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe `List` request returns an array containing all data registries previously registered using the [Register or Replace API](https://docs.microsoft.com/rest/api/maps/data-registry/register-or-replace).\n\nThis API won't check whether Azure Maps can access the underlying content and whether integrity check is passed for each data registry. Please use [Get Content](https://docs.microsoft.com/rest/api/maps/data-registry/get-content) for the latest status individually.\n\nThe data registry service endpoint is limited in scope to the region of your Azure Maps account and is not available for global Azure Maps accounts.\n\n## Submit List Request\n\nTo list all your data registries, issue a `GET` request with no additional parameters.\n\n## List Response\n\nThe `List` request returns an array of data registries in `JSON` format in the body of the response.",
```

We no long check the underlying content in Get Data Registry API. Users need to use Get Content for this purpose.